lib/mongodb.js

```diff
@@ -6,6 +6,7 @@ var util = require('util');
 var async = require('async');
 var Connector = require('loopback-connector').Connector;
 var debug = require('debug')('loopback:connector:mongodb');
+var GeoPoint = require('loopback-datasource-juggler/lib/geo').GeoPoint;
```

Comment:
`loopback-datasource-juggler` may not be require-able from the connector. Can we use a different approach to detect GeoPoint properties? E.g. 

```js
if(prop && prop.type && prop.type.name === 'GeoPoint')
```